pkg/diagnostics/diagnostic_bundle.go

```diff
@@ -142,7 +144,10 @@ func newDiagnosticBundleDefault(af AnalyzerFactory, cf CollectorFactory) *EksaDi
 		analyzerFactory:  af,
 		collectorFactory: cf,
 	}
-	return b.WithDefaultAnalyzers().WithDefaultCollectors().WithManagementCluster(true)
+	return b.WithDefaultAnalyzers().
+		WithDefaultCollectors().
+		WithFileCollectors([]string{logger.GetOutputFileName()}).
```

Comment:
how does this work when the bundle is not generated from a `cluster` command? It could be generated directly with the `generate supportbundle` or whatever command. In that case, we probably don't need this.